tests/integration/identity.go

```diff
@@ -21,31 +21,59 @@ import (
 	acpIdentity "github.com/sourcenetwork/defradb/acp/identity"
 )
 
-// identityRef is a type that refers to a specific identity of a certain type.
-type identityRef struct {
-	isClient bool
-	index    int
+type identityType int
+
+const (
+	noType identityType = iota
+	clientType
+	nodeType
```

Comment:
todo: please add some scope prefix, because `integration` package is somewhat big and from other context these consts might be confusing.